.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I also sometimes receive some settings.json but decide not to commit, but this is fine, right?